cmd/mattermost/commands/teamargs.go

```diff
@@ -19,15 +19,12 @@ func getTeamsFromTeamArgs(a *app.App, teamArgs []string) []*model.Team {
 
 func getTeamFromTeamArg(a *app.App, teamArg string) *model.Team {
 	var team *model.Team
-	if result := <-a.Srv.Store.Team().GetByName(teamArg); result.Err == nil {
-		team = result.Data.(*model.Team)
-	}
+	team, err := a.Srv.Store.Team().GetByName(teamArg)
 
 	if team == nil {
-		if t, err := a.Srv.Store.Team().Get(teamArg); err == nil {
+		if t, _ := a.Srv.Store.Team().Get(teamArg); err == nil {
```

Comment:
you are changing here the behavior, you are removing the assignation of the `err` returned by the store here, and reusing the previous error, which would introduce a bug. if this is for avoid "shadowed variables" you should just rename the error or declare previously the `t` variable and use `=`  instead of `:=`.